### ISSUE TITLE
Problem publishing a page

### ISSUE THREAD

**Issue by Wirzi** [divio/django-cms#4937]
I get this error when i try to publish a page.
Tested django-cms versions: 3.1.3, 3.1.4, 3.2

```
Internal Server Error: /rm/admin/cms/page/285/rm/publish/
Traceback (most recent call last):
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 233, in inner
    return view(request, *args, **kwargs)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/django/utils/decorators.py", line 34, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/django/views/decorators/http.py", line 45, in inner
    return func(request, *args, **kwargs)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/django/utils/decorators.py", line 30, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/django/utils/decorators.py", line 145, in inner
    return func(*args, **kwargs)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/reversion/revisions.py", line 297, in do_revision_context
    return func(*args, **kwargs)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/cms/admin/pageadmin.py", line 973, in publish_page
    published = page.publish(language)
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/cms/models/pagemodel.py", line 669, in publish
    if page.publisher_public.parent.is_published(language):
  File "/home/vagrant/.virtualenvs/example/local/lib/python2.7/site-packages/cms/models/pagemodel.py", line 536, in is_published
    return self.get_title_obj(language, False, force_reload=force_reload).published
AttributeError: 'NoneType' object has no attribute 'published'
```

Any idea why?

*Edit*:
I don't know if it has something to do with it, but i used `python manage.py cms copy-lang de rm` before.

**Comment by czpython** [comment]
@Wirzi Hi there,

You mentioned you can reproduce on 3.1.3, 3.1.4, 3.2.
Did you run ``python manage.py cms copy-lang de rm`` on each version as well?

**Comment by Wirzi** [comment]
@czpython No only on 3.1.3 and upgraded afterwards.

**Comment by czpython** [comment]
This is fixed on 3.3
Please run ``./manage.py cms fix-tree`` before upgrading.
